sipa/templates/usersuite/index.html

```diff
@@ -13,17 +13,18 @@
 {% endblock %}
 
 {% block sidebar %}
-    <nav id="usersuite-sidebar-nav" class="js-toc toc d-none d-lg-block"></nav>
-{% endblock %}
-
-{% block custom_css %}
-    <link rel="stylesheet" href="{{ url_for("static", filename="css/tocbot.css") }}"/>
-    <link rel="stylesheet" href="{{ url_for("static", filename="css/usersuite_index.css") }}"/>
+    <nav class="nav nav-pills sticky-top d-none d-lg-block">
+        <a class="nav-link" href="#hot-buttons">{{ _("Häufige Aktionen") }}</a>
+        <a class="nav-link" href="#status-information">{{ _("Statusinformationen") }}</a>
+        <a class="nav-link" href="#payment-details">{{ _("Zahlungsdetails") }}</a>
+        <a class="nav-link" href="#traffic">{{ _('Dein Traffic') }}</a>
+        <a class="nav-link ms-3" href="#traffic-table">{{ _('Traffic Tabelle') }}</a>
+        <a class="nav-link ms-3" href="#traffic-chart">{{ _('Traffic Diagramm') }}</a>
+        <a class="nav-link" href="#transaction-log">{{ _('Buchungen') }}</a>
+    </nav>
```

Comment:
I'm really sorry, but I have to reject this. You're making things objectively worse here. This is redundant information that will go out of sync the minute someone touches the usersuite headings, not least because they are defined in a separate file. Tocbot did things right by discovering the structure of a given DOM-subtree itself (and tbh has a slicker design by hiding sub-headings).

While I guess it would be fine if you added a dynamic TOC discovery to this bootstrap-based solution, I don't quite see the point of that. Sure, in JS it's kind of a cliche to depend on libraries that are more baggage than they are worth, but I think you're unfair to tocbot here:

- tocbot has _zero_ dependencies
- solves an actual problem and delivers some nontrivial features (heading discovery)
- while being vanilla JS and thus pretty small (11kb; gzipped only 3.5 – to contrast this, my usersuite is ~85kB large)
- Is unlikely to be a security risk, precisely due to the small size/amount of dependencies/specific nature of the tool
 
Sipa would profit much more from proper performant static file delivery (+ gzipping the libraries, see #441) than from re-solving a solved problem.

Reply:
Sorry that I didn't have a chance to review your PR earlier and give you more immediate feedback. I looked at parts before but couldn't look at the whole thing.

Reply:
One advantage of using Bootstrap here is that it integrates better UI-wise but I acknowledge the arguments against this change.